<launch>
    <node name="controller_node" type="controller" pkg="inverted_pendulum_pkg" output="screen" />
</launch>